src/app/scripts/cac/routing/cac-routing-plans.js

```diff
@@ -25,7 +23,7 @@ CAC.Routing.Plans = (function($, L, moment, _, UserPreferences, Itinerary) {
     function planTrip(coordsFrom, coordsTo, when, mode, arriveBy) {
         var deferred = $.Deferred();
         var urlParams = prepareParamString(coordsFrom, coordsTo, when, mode, arriveBy);
-        var requestUrl = routingUrl + '?' + urlParams;
+        var requestUrl = Settings.routingUrl + '?' + urlParams;
```

Comment:
I know what you're working on doesn't touch the formatting part, but I just noticed that perhaps it would be better to pass the parameters object directly to the `$.ajax` call's `data` argument, to let it do the URL formatting, rather than using a combination of `prepareParamString` invoking `$.param`, and then having to concatenate with the `?` here.